### PR TITLE
PS-7270: extra_port should not accept proxied connections - 8.0

### PR DESCRIPTION
**Pull request by imp3142**

### CODE REVIEW

**Comment by altmannmarcelo** [comment]
https://ps80.cd.percona.com/job/percona-server-8.0-param/750/

File: mysql-test/t/proxy_protocol_admin_port.test [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+--source include/count_sessions.inc
+
+--let $PORT_OFFSET = 100
+--expr $ADMIN_PORT = $MASTER_MYPORT + $PORT_OFFSET
+--let $restart_parameters=restart: --skip-name-resolve --admin-address=::ffff:127.0.0.1 --admin_port=$ADMIN_PORT --proxy_protocol_networks=*
+--replace_result $ADMIN_PORT ADMIN_PORT
+--source include/restart_mysqld.inc
+
+--echo # An unproxied client connection should be accepted when connection to an extra_port if proxying enabled on the server
+--disable_query_log
```
Comment (percona-ysorokin) [inline mysql-test/t/proxy_protocol_admin_port.test]: Either remove this or add corresponding `--disable_query_log` afterwards

File: mysql-test/t/proxy_protocol_admin_port.test [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+connect (conn1,"127.0.0.1",root,,,$ADMIN_PORT,);
+--connection conn1
+SHOW GLOBAL STATUS LIKE 'Threads_connected';
+
+--source include/remove_extra_root_users.inc
```
Comment (percona-ysorokin) [inline mysql-test/t/proxy_protocol_admin_port.test]: move this after
```
--disconnect conn1
--connection default
```
to make sure we are on `default` connection

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Comment by altmannmarcelo** [comment]
https://ps80.cd.percona.com/job/percona-server-8.0-param/761/ - `proxy_protocol_admin_port` test is passing on all platforms.

**Pull request merged**